tedana/metrics/kundu_fit.py

```diff
@@ -107,7 +107,7 @@ def dependence_metrics(catd, tsoc, mmix, t2s, tes, ref_img,
     WTS = computefeats2(tsoc, mmixN, mask=None, normalize=False)
 
     # compute PSC dataset - shouldn't have to refit data
-    tsoc_B = get_coeffs(tsoc_dm, mmix, mask=None)
+    tsoc_B = get_coeffs(tsoc_dm, mmix, mask=None, add_const=False)
```

Comment:
Are we adding the last two arguments just for clarity ? We shouldn't need either of them, actually, since they're the defaults !

Reply:
Sorry for the delay. Personally, I prefer to have the default option indicated in the call because it is easier to understand/read if you don't remember it.